README.rst

```diff
@@ -380,14 +380,47 @@ logins on the same device, you do not wish the old user to receive messages whil
 Via DRF, any creation of device with an already existing registration ID will be transformed into an update.
 If done manually, you are responsible for deleting the old device entry.
 
+Using custom FCMDevice model
+----------------------------
+
+If there's a need to store additional information or change type of fields in the FCMDevice model.
+You could simple override this model. It could be fully your own model or you could inheriting from AbstractFCMDevice 
+
+In your ``your_app/models.py``:
+
+.. code-block:: python
+
+    import uuid
+    from django.db import models
+    from fcm_django.models import AbstractFCMDevice
+
+
+    class CustomDevice(AbstractFCMDevice, models.Model):
+        # fields could be overwritten
+        id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+        # could be added new fields
+        updated_at = models.DateTimeField(auto_now=True)
+
+In your ``settings.py``:
+
+.. code-block:: python
+
+    FCM_DJANGO_FCMDEVICE_MODEL = "your_app.CustomDevice"
+
+Note: This functionality based on `Swapper <https://pypi.org/project/swapper/>`_ that based on functionality 
+that allow to use a `custom User model <https://docs.djangoproject.com/en/4.2/topics/auth/customizing/#substituting-a-custom-user-model>`_.
+So this functionality have the same limitations. 
+The most is important limitation it is that is difficult to start out with a default (non-swapped) model 
+and then later to switch to a swapped implementation without doing some migration hacking.
+
 Python 3 support
 ----------------
 - ``fcm-django`` is fully compatible with Python 3.7+
 - for Python 3.6, use ``fcm-django < 2.0.0`` , because `firebase-admin with version 6 drop support of Python 3.6 <https://firebase.google.com/support/release-notes/admin/python#version_600_-_06_october_2022>`_
 
 Django version compatibility
 ----------------------------
-Compatible with Django versions 3.0+.
+Compatible with Django versions 3.2+.
```

Comment:
Is this needed? Swapper appears to be compatible with Django 2+

Reply:
Django 2.0 supports Python 3.4, 3.5, 3.6, and 3.7.
And [there's no support for Django 2.0](https://www.djangoproject.com/download/) on official page.
I though that we wont support releases that aren't have official security updates it could be great to update docs. Ie we drop support of Python <=3.6

But I can rollback those changes.

Reply:
We will support whatever works with the current setup, we shouldn't be as optinionated as to drop support just because something does not have security updates :-)

Reply:
If so why was support of python3.6 dropped?

Reply:
Because firebase-admin does not support python 3.6 in their latest release. I'm not saying we should make an effort to support django 2, we should merely not remove it from compatible packages if it works

Reply:
> Because firebase-admin does not support python 3.6 in their latest release.
But that doesn't mean that it wont works with current version of package. They removed it because python 3.6 become deprecated.

Reply:
Unsupported by who?

Reply:
> Unsupported by who?

by Django core team.

Reply:
I rollback this change